runtime/moonriver/src/lib.rs

```diff
@@ -813,6 +813,10 @@ impl Get<Slot> for RelayChainSlotProvider {
 	}
 }
 
+parameter_types! {
+	pub const LinearInflationThreshold: Option<Balance> = Some(1_200_000_000 * currency::MOVR);
+}
+
```

Comment:
Important to note that moonbeam supply factor is 100 times the supply of moonriver and moonbase. This will probably never be reached